contents/docs/user-guides/funnels.md

```diff
@@ -4,74 +4,175 @@ sidebar: Docs
+If you’re measuring something recurrently  or you’d like to share it with others or easily return to it again in the future, hit the “disk” icon in the top left of the configuration pane.
 
-## Important note on funnels
+**Step 2: Naming funnel and selecting dashboard**
 
-It is important to note that funnels calculate the drop off percentage between steps based on unique users *only*.
+Select a dashboard where you would like to save the funnel and give it a name
 
-This means "action chains" are not calculated individually, but rather compiled based on a user's entire set of actions.
+**Step 3: Finding a saved funnel**
 
-For example, if User X did Step 1 of a certain funnel 5 times, but then only did Step 2 a single time, this will still be shown as 100% conversion between the steps, instead of 20%. 
+You can find your funnel any time by navigating to the dashboard you’ve saved it in.
 
-This happens because the conversion on that step for the user was 100%, even if the *individual conversion per action* was not.
+Recently saved funnels are also available within the funnel configuration screen.
+
+### Modifying a funnel
+
+Once you have your funnel setup, you may like to tweak the step ordering or filters, this is easy. Steps can be dragged around by clicking and holding down the icon to the left of the step number in the configuration area of the screen.
+
+## Analyzing Funnels
+
+
+### Understand where people are getting stuck during your flow
+
+The most common use case for funnels is to understand where people are getting stuck or dropping off in your flow.
+
+![image](https://user-images.githubusercontent.com/85295485/134479685-35702219-96e2-497e-a837-04f999fead00.png)
+
+Quickly looking at this funnel we can see there are a number of drop-offs between steps. Absolute drop-off is a great way to understand where you’re losing the most people, but if you want to identify the areas which are having the greatest negative impact on your overall conversion rate - you’ll want to understand the relative drop-off. Switch to relative drop-off by clicking “% Overall conversion” and switching to “% Relative to previous step”
+
+![image](https://user-images.githubusercontent.com/85295485/134480083-6cf799ba-8720-43e5-a88a-bc1fcaa338f3.png)
+
+You can use this to see where the biggest opportunity for improvement is, usually on the step with the lowest relative conversion rate.
+
+
+### Find out who is successful and who is not
+
+Once you have a funnel drop-off you’d like to explore further, the first step is to identify whether it’s a specific client or users that are having the problem and find out why they were struggling. There are many ways to get this information, from reaching out to them directly to [replaying their session](/docs/user-guides/sessions).
+
+Click on the number next to the drop-off to see the people who did not successfully complete this step of your funnel.
+
+[Screenshot of people modal from step drop-off]
```

Comment:
@marcushyett-ph could you please provide this image?

I tried finding a funnel and clicking on the "Dropped" number (1719) but no modal appeared.

![image](https://user-images.githubusercontent.com/328367/134551498-56382283-2c8c-4d97-a2ca-d7df762fed62.png)

Reply:
Feels like a bug.... @alexkim205 have you seen this?

Reply:
Hmm definitely looks like a bug, @leggetter do you happen to have the link where this happened?

Reply:
Try [this link](https://app.posthog.com/insights?insight=FUNNELS&date_from=-30d&display=FunnelViz&actions=%5B%7B%22id%22%3A5043%2C%22type%22%3A%22actions%22%2C%22order%22%3A0%2C%22name%22%3A%22App%20Pageview%20-%20Logged%20in%22%2C%22math%22%3A%22dau%22%2C%22math_property%22%3Anull%2C%22properties%22%3A%5B%7B%22key%22%3A%22%24active_feature_flags%22%2C%22value%22%3A%224050-query-ui-optB%22%2C%22operator%22%3A%22regex%22%2C%22type%22%3A%22event%22%7D%5D%7D%2C%7B%22id%22%3A4912%2C%22type%22%3A%22actions%22%2C%22order%22%3A1%2C%22name%22%3A%22Discovered%20Learning%22%2C%22math%22%3Anull%2C%22math_property%22%3Anull%2C%22properties%22%3A%5B%5D%7D%5D&filter_test_accounts=true&funnel_window_days=14&interval=day&funnel_viz_type=steps&funnel_from_step=0&funnel_to_step=1&hiddenLegendKeys=%7B%22actions%2F5043%2F0%22%3Atrue%2C%22actions%2F4912%2F1%22%3Atrue%7D&breakdown=%24os&breakdown_type=person#fromItem=219560&fromItemName=App%20Usage%20to%20Discover%20Learning%20-%20Experiment&fromDashboard=3479).

However, when you click on another action button the modal opens. It's like the open action is queued.

https://user-images.githubusercontent.com/328367/134556167-f2e5b1b9-23de-4d6b-8793-f0284e8f3cd8.mp4

Reply:
Ah I think I know why. Person modals are disabled in "Dashboard mode" and a saved insight is considered a saved dashboard item. Made a fix here #6056